orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/AuthCodeResponseGenerationService.java

```diff
@@ -143,7 +143,9 @@ public void saveSession(
             sessionService.storeOrUpdateSession(
                     session.setAuthenticated(true).setNewAccount(EXISTING));
             orchSessionService.updateSession(
-                    orchSession.withAccountState(OrchSessionItem.AccountState.EXISTING));
+                    orchSession
+                            .withIsAuthenticated(true)
```

Comment:
Looks like we don't have tests for this service..?

Reply:
Have added some test in the latest commit